### DISCUSSION TITLE
Create mock server from spec

### DISCUSSION THREAD

**Discussion by namp10010** [oapi-codegen/oapi-codegen#723] [Ideas]
Sorry if this has been asked before but I couldn't find the documentation to generate mock server from spec.

Is it currently available?

If not, does it make sense to have that feature to essentially create a ready to use mock server with examples directly from the spec?

**Comment by jamietanna** [comment]
Hey @namp10010 I guess you mean a mock server that implements the OpenAPI spec and can provide realistic data in its responses?

[I've written about this before](https://www.jvt.me/posts/2022/03/04/openapi-local-prism/) and would recommend [Prism](https://github.com/stoplightio/prism/) as a tool that can do this beautifully!

**Reply by namp10010** [reply]
Hi @jamietanna , thanks a lot for sharing the tool. It looks pretty cool, I will give it a try.
